### ISSUE TITLE
Download as .EML / Show source not working

### ISSUE THREAD

**Issue by nichtmax** [the-djmaze/snappymail#1858]
**Describe the bug**
Download as .EML or show source results in an error: Access denied / Disallowed Sec-Fetch Dest: document Mode: navigate Site: cross-site User: false

**To Reproduce**
1. Go to a specific mail
2. Click on Download as .EML / Show source
3. See error

**Expected behavior**
Download as .EML starts / source is shown

**Screenshots**
![Screenshot 2024-12-02 at 15 04 13](https://github.com/user-attachments/assets/d3ff8acd-7bf5-4e76-8a74-e9db5fbf9b24)

**Please complete the following information:**
- Browser: Safari / Version 18.1.1 (20619.2.8.11.12)
- IMAP daemon: dovecot
- PHP version: 8.1
- SnappyMail Version: 2.38.2
- Mode: standalone

**Additional context**
We're pretty sure this is a new behavior which is introduced with the update from 2.38.1

**Comment by the-djmaze** [comment]
My main concern is that Safari thinks it is "cross-site".
Did you test with other browsers?

**Comment by nichtmax** [comment]
You're right, this seems to be a Safari only problem. It's working in Firefox and Chrome.
